test/models/SoftwareProviderModelSpec.scala

```diff
@@ -79,5 +75,13 @@ class SoftwareProviderModelSpec extends TestUtils {
 
       actualResult shouldBe expectedResult
     }
+
+    "read from Json correctly with all filters" in {
```

Comment:
Technically, all filters with the exception of Welsh!

Reply:
I have added Welsh to the list of filters in the test.